### PR TITLE
Mamoru 83 integrate mamoru core to bsc light node

### PR DESCRIPTION
**Pull request by gofmanaa**
MAMORU-83: Integrate Mamoru Core to BSC Light node
- Create a new GitHub workspace
- Create a tracer for the BSC light chain

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: "Build docker image"
```
Comment (sokyrko) [inline .github/workflows/docker.yml:1]: Please check if it works (you may modify the workflow to trigger the image build from this branch)

File: .github/workflows/releas.yml [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+      # ==============================
+      - name: Build Binary for ${{matrix.os}}
+        run: make geth
+
+      - name: Upload Linux Build
+        uses: actions/upload-artifact@v2
+        if: matrix.os == 'ubuntu-22.04'
+        with:
+          name: linux
+          path: ./build/bin/geth
```
Comment (sokyrko) [inline .github/workflows/releas.yml:48]: What is `geth`? Do we need it?

**Review by sokyrko — APPROVED** [review]

File: .github/workflows/releas.yml (hunk 2 above)
Comment (gofmanaa) [reply .github/workflows/releas.yml:48]: This is the main Ethereum CLI client.  It is the entry point into the Ethereum network.

File: .github/workflows/docker.yml (hunk 1 above)
Comment (gofmanaa) [reply .github/workflows/docker.yml:1]: checked

**Pull request merged**